.env

```diff
@@ -0,0 +1,5 @@
+DB_USERNAME='bruins'
+DB_PASSWORD='Bruin$NHL'
+DB_DATABASE='StanleyCupFinals'
+DB_HOST='localhost'
+DB_DIALECT='mysql'
```

Comment:
Once you've committed it adding it to .gitignore won't work. You have to delete it. Commit that deletion. Then you can add it to .gitignore and when you recreate the file it will start ignoring it.